webservices/WFLY-13356_EAP7-1525.adoc

```diff
@@ -0,0 +1,51 @@
+= WFLY-13356/ EAP7-1525 Add WebServices Galloen layer
+:author:            Jim Ma
+:email:             ema@redhat.com
+:toc:               left
+:icons:             font
+:keywords:          comma,separated,tags
+:idprefix:
+:idseparator:       -
+:issue-base-url:    https://issues.jboss.org/browse
+
+== Overview
+
+
+
+== Issue Metadata
+
+=== Issue:
+
+* {issue-base-url}/EAP7-1525[EAP7-1525]
+* {issue-base-url}/WFLY-13356[WFLY-13356]
+
+=== Related Issues:
+
+* {issue-base-url}/WFLY-13354[WFLY-13354]
+* {issue-base-url}/WFLY-13771[WFLY-13771]
+
+=== Dev Contacts:
+
+* mailto:ema@redhat.com[Jim Ma]
+
+=== QE Contacts:
```

Comment:
The "tested by" section is missing (this is going to be tested by engineering, right?)